installers/ansible/inventory

```diff
@@ -7,6 +7,10 @@ crunchy_debug='false'
 # ==================
 # Kubernetes and OpenShift Settings
 # Note: Either Kubernetes or OpenShift must be used
+# One of the three following blocks must be updated:
+# - Deploy into Openshift
+# - Deploy into Kubernetes
+# - Deploy with PGO-Installer
```

Comment:
`# - Deploy with the PostgreSQL Operator Installer (pgo-installer)`

and make this one the first one.

Provide a bit more explanation about the differences, i.e. the Installer is a container that runs within your Kubernetes/OpenShift environment, whereas the other ones allow you to run the installer from your local environment into the remote Kube/OpenShift ones